src/main/fc/fc_tasks.c

```diff
@@ -50,10 +49,12 @@
 #include "fc/cli.h"
 #include "fc/fc_dispatch.h"
 
+#ifdef USE_FC
```

Comment:
it wasn't to do with the build time saving, it was to do with FC specific #defines being needed.